convert_to_BIDS.py

```diff
@@ -1,31 +1,116 @@
+        os.mkdir(rootBIDSDataPath)
+
+    img_dict = {'_T1_manual_lesion_label.nii.gz':('_T1w_seg-tumor.nii.gz','derivatives/labels'),
+                '_T2_manual_lesion_label.nii.gz':('_T2w_seg-tumor.nii.gz','derivatives/labels'),
+                '_T1w.nii.gz':('_T1w.nii.gz',''),
+                '_T2w.nii.gz':('_T2w.nii.gz',''),
+                }
+    #Rearrange data into BIDS format
+    for cwd, dirs, files in os.walk(rootDataPath):
+        for file in files:
+            file_path = os.path.join(cwd,file)
+            
+            for img_key in img_dict.keys():
+                if file.endswith(img_key):
+                    sub_full = str(file.split('_')[0])
+                    sub_id = (re.findall('\d+', sub_full ))
+                    sub_type = cwd.split('/')[-3].split('_')[0][0:4]
+                    sub_id_new = "sub-"+ sub_type + sub_id[0]
+                    sub_new_folder = os.path.join(rootBIDSDataPath,img_dict[img_key][1],sub_id_new , "anat")
+                    if not os.path.exists(sub_new_folder):
+                        os.makedirs(sub_new_folder)
+                    new_file_path = os.path.join(sub_new_folder, sub_id_new + img_dict[img_key][0])
+                    print ("Initial path: " + file_path)
+                    print ("New path:     " + new_file_path + '\n')
+                    shutil.copy(file_path,new_file_path)
+
+    #Create participants.json
+    content_participants_json = [
+            {
+                "participant_id": {
+                    "LongName": "Participant ID",
+                    "Description": "Unique ID"
+                },
+                "sex": {
+                    "LongName": "Participant gender",
+                    "Description": "M or F"
+                },
+                "age": {
+                    "LongName": "Participant age",
+                    "Description": "yy"
+                }
+                }]
+    # Save participants.json
+    with open(participants_jsonFILENAME, 'w') as outfile:
+        outfile.write(json.dumps(content_participants_json[0], indent=2))
+        outfile.close()
+
+    #Create dataset_description.json
+    dataset_description = {}
+    dataset_description[u'Name'] = 'SCT_Tumor'
+    dataset_description[u'BIDSVersion'] = '1.2.1'
+
+    # Save dataset_description.json
+    with open(dataset_description_jsonFILENAME, 'w') as outfile:
+        outfile.write(json.dumps(dataset_description,
+                                    indent=2, sort_keys=True))
+        outfile.close()
+    participants =[]
+    #Create participants.tsv
+    for dirs in os.listdir(rootBIDSDataPath):
+        if dirs.startswith('sub-'):
+            row_participants = []
+            row_participants.append(dirs)
+            row_participants.append('-')
+            row_participants.append('-')
+            participants.append(row_participants)
+            
+    # # Save participants.tsv
+    with open(participants_tsvFILENAME, 'w') as tsv_file:
```

Comment:
Would it be preferable to add the tumor type in the participants.tsv file instead?

Reply:
Fixed in https://github.com/sct-pipeline/tumor-segmentation/pull/1/commits/d260a0353b2d9df9bc60419ce0888a64905984de